Atomic/sign.py

```diff
@@ -0,0 +1,107 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+import re
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # Atomic is run as sudo. Should we work around that?
+
+        if self.args.debug:
+            util.write_out(str(self.args))
+
+        # Verify all the images provided are valid
+        self.check_input_validity()
+
+        signature_path = util.get_signature_path(self.atomic_config)
+        use_def_sig_path = True if signature_path == self.args.output else False
+
+        signer = self.args.sign_key
+
+        for sign_image in self.args.images:
+            remote_inspect_info = util.skopeo_inspect("docker://" + sign_image)
+            manifest = util.skopeo_inspect('docker://{}'.format(sign_image), args=['--raw'], return_json=False)
```

Comment:
One line uses `+`, the line immediately below uses `'{}'.format`. That doesn’t really _hurt_ but having both consistent would make it clearer that nothing is going on here.